### DISCUSSION TITLE
How can I make the data for confirmed and deaths fit better with the model?

### DISCUSSION THREAD

**Discussion by eabase** [neherlab/covid19_scenarios#428] [General]
I'm looking at the Swedish data and I am not able to fit both the cases and the deaths at the same time. Those model curves are too far apart. So how can I better model by narrowing those model curves? I.e. what parameters do i need to adjust?

**Comment by noleti** [comment]
Most likely, the reported cases are underestimating real infected. Have a look at the app development previews, e.g. https://covid19-scenarios-qwqgpydh5.now.sh/
There is some automated fitting based on deaths done there, with some (made-up I believe) entries for mitigations. I manually added a mitigation of 0.5 from March 17 onwards, which fits ok in my opinion.

**Reply by deuteragenie** [reply]
Indeed, automatic fitting to death cases is a much more reliable predictor. Is there a chance that this feature makes it to the main site ?

**Reply by ivan-aksamentov** [reply]
@deuteragenie This is now released on the new website:
https://covid19-scenarios.org/

The latest features (prerelease) are on master branch:
https://master.covid19-scenarios.org/

You can observe and/or participate in the development in our
issues: https://github.com/neherlab/covid19_scenarios/issues
pull-requests: https://github.com/neherlab/covid19_scenarios/pulls

See also Richard's tweet:
https://twitter.com/richardneher/status/1248000209608749057

**Comment by nnoll** [comment]
On the development branch we have automated this process for you. Currently, we fit to both deaths and cases, however only after estimating the "under-reported" fraction for each region. This will most definitely make it to the website, and we hope the release will be in the next few days.

**Reply by deuteragenie** [reply]
Great thanks !

It would be good to have several auto-fitting options on the site: a) Fit to cases b) Fit to deaths c) Autofit to both etc.
